SpatialGDK/Source/SpatialGDK/Private/EngineClasses/SpatialNetDriver.cpp

```diff
@@ -388,6 +388,15 @@ void USpatialNetDriver::CreateAndInitializeCoreClasses()
 	if (SpatialSettings->UseRPCRingBuffer())
 	{
 		RPCService = MakeUnique<SpatialGDK::SpatialRPCService>(ExtractRPCDelegate::CreateUObject(Receiver, &USpatialReceiver::OnExtractIncomingRPC), StaticComponentView);
+
+		SpatialGDK::EPushRPCResult PushRPCFailureOutcomesToQueue = SpatialGDK::EPushRPCResult::Overflowed | SpatialGDK::EPushRPCResult::AlreadyQueued;
+		// Clients and single server deployments can also safely queue (on the sender) RPCs that fail because of NoRingBufferAuthority, since authority over the relevant endpoint is guaranteed to be received
```

Comment:
This is only true for controllers.

Reply:
Hmm could be a misunderstanding on my part then.  There are (non-controller) cases where the client will not end up with authority over the client endpoint component?  I thought this was guaranteed via the WriteACL:

ComponentWriteAcl.Add(SpatialConstants::CLIENT_ENDPOINT_COMPONENT_ID, OwningClientOnlyRequirementSet);

Reply:
Similar Q for server endpoint in a single server scenario:

ComponentWriteAcl.Add(SpatialConstants::SERVER_ENDPOINT_COMPONENT_ID, AuthoritativeWorkerRequirementSet);

Reply:
I see that for the PC, the ReadACL is set to AnyServerOrOwningClientRequirementSet, whereas for other entities it's set to AnyServerOrClientRequirementSet.  Is this the concern?  Ie. A different client may try to call an RPC on a client endpoint they will never own, but have read access to?

Reply:
I'll put up the alternate solution of moving the RPC call in a draft PR today.